tools/build_defs/configure_script.bzl

```diff
@@ -34,27 +35,50 @@ def create_configure_script(
     )]
     return "\n".join(script)
 
-def _define_deps_flags(inputs):
-    libs = collect_libs(inputs.deps_linking_info)
-
+def _define_deps_flags(deps, inputs):
     # It is very important to keep the order for the linker => put them into list
     lib_dirs = []
 
-    # For filtering duplicates
+    # Here go libraries built with Bazel
     lib_dirs_set = {}
-    for lib in libs:
+    for lib in inputs.libs:
         dir = lib.dirname
         if not lib_dirs_set.get(dir):
             lib_dirs_set[dir] = 1
-            lib_dirs += [dir]
+            lib_dirs += ["-L$EXT_BUILD_ROOT/" + dir]
+
+    include_dirs_set = {}
+    for list in inputs.include_dirs:
```

Comment:
Rename list to a more specific name and also one that doesn't shadow the starlark "list"